### ISSUE TITLE
[Bug]: Inconsistent documentation versus behavior

### ISSUE THREAD

**Issue by pencilcheck** [coollabsio/coolify#4036]
### Error Message and Logs

https://github.com/coollabsio/coolify/issues/2443

The issue above didn't reflect that in the official documentation so it is very confusing what is the actual behvaior of the env of the app it is using unless you go to deployment and inspect the env being passed into the build.

Please update so both are consistent, coming from outside I have no clue someone swapped it and it messed up a lot of the build

### Steps to Reproduce

1.
2.
3.
4.

### Coolify Version

v4.0.0-beta360

### Are you using Coolify Cloud?

No (self-hosted)

**Comment by peaklabs-dev** [comment]
Yes, that is true, I am planning a refactor for this and the other Magic Envs to make them much better.

**Comment by andrasbacsai** [comment]
Closing old issue (which is probably fixed already). Reopen if still reproducible.
